specification.md

```diff
@@ -196,6 +211,28 @@ Represents person registration data.
 }
 ```
 
+### RegistrationInfo
+
+Represents information related to when and how to register for the competition.
+
+| Attribute | Type | Description |
+| --- | --- | --- |
+| `registrationOpen` | [`DateTime`](#datetime) | The point in time when online registration opens. |
+| `registrationClose` | [`DateTime`](#datetime) | The point in time when online registration closes. |
+| `baseEntryFee` | `Integer` | The competition's base fee for online registration. |
+| `currencyCode` | [`CurrencyCode`](#currencycode) | The currency of the `baseEntryFee`. |
+
+#### Example
+
+```json
+{
+  "registrationOpen": "2023-08-29T05:00:00Z",
```

Comment:
I don't really like the idea of doing `.registrationInfo.registrationOpen`. That names are redundant. We could have just `.registration.openDate` or for consistency, `.registration.startDate`